.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: python
+python:
+  - "2.7"
+  - "3.4"
+  - "3.5"
+os:
+  - linux
+# ** patchelf isn't supported here, _yet_ **
+# addons:
+#   apt:
+#     sources:
+#       - ppa:utlemming/rax-playground
+#     packages:
+#       - build-essential
+#       - ninja-build
+#       - patchelf
+#       - libboost-dev
+#       - libboost-program-options-dev
+#       - libogg-dev
```

Comment:
I'm not sure I understand these comments. Are they just documentation of the lines after it, or is this something that should get uncommented?